### ISSUE TITLE
Connection parameters can only be passed on the command line

### ISSUE THREAD

**Issue by mraspaud** [pytroll/trollflow2#198]
## Feature Request

**Is your feature request related to a problem? Please describe.**
At the moment, it it not possible to pass the address to listen to in the configuration file (product list). Instead, the address can only be configured with the `-a` flag on the command line of the satpy laucher.

**Describe the solution you'd like**
I would like to be able to have the connection parameters in the config file.

**Describe any changes to existing user workflow**
That will allow to have static scripts/entrypoints to launch trollflow2 and pass the address to connect to only in the config file.

**Comment by pnuu** [comment]
I've used an environment variable in my container entrypoint and have injected it when starting the container to work around this. I don't remember why I didn't search for a better solution by changing the actual code.

**Comment by mraspaud** [comment]
Yes, I know. But even passing a config without a `loggers` section fails.
